shared-bindings/_bleio/PacketBuffer.c

```diff
@@ -168,31 +183,72 @@ STATIC MP_DEFINE_CONST_FUN_OBJ_1(bleio_packet_buffer_deinit_obj, bleio_packet_bu
 
 //|   .. attribute:: packet_size
 //|
-//|     Maximum size of each packet in bytes. This is the minimum of the Characteristic length and
-//|     the negotiated Maximum Transfer Unit (MTU).
+//|     `packet_size` is the same as `incoming_packet_length`.
+//|     The name `packet_size` is deprecated and
+//|     will be removed in CircuitPython 6.0.0.
+//|
+//|   .. attribute:: incoming_packet_length
+//|
+//|     Maximum length in bytes of a packet we are reading.
+//|     If the packet is arriving from a remote service via notify or indicate,
+//|     the maximum length is `Connection.max_packet_length`.
+//|     Otherwise it is the ``max_length`` of the :py:class:`~_bleio.Characteristic`.
```

Comment:
I feel like this is an implementation detail that we shouldn't reveal here. (Same below.)

Reply:
OK, though I think it will get explained to someone at some point.